frontend/enums/SetupScreen.ts

```diff
@@ -10,7 +10,7 @@ export const SetupScreen = {
   FundYourAgent: 'FundYourAgent',
   TransferFunds: 'TransferFunds',
   SetupBridgeOnboardingScreen: 'SetupBridgeOnboardingScreen',
-  SetupOnRamp: 'SetupOnRamp',
+  OnRamp: 'OnRamp',
```

Comment:
confused now, so we keep using OnRamp page for setupScreen? it's been removed from `frontend/components/SetupPage/index.tsx`